### ISSUE TITLE
how to define a reshape layer?

### ISSUE THREAD

**Issue by Equationliu** [locuslab/convex_adversarial#26]
I have defined a network to train the mnist datasets, and it works well when use my own loss, but when considering the robust_loss in convex_adversarial, there is an error

`: No module for layer Reshape`

following is my code:

```
# reshape method for Sequential
class Reshape(nn.Module):
    def __init__(self, *args):
        super(Reshape, self).__init__()
        self.shape = args
    def forward(self, x):
        return x.view(self.shape)

# Sequential model
Net = nn.Sequential(
          nn.Conv2d(1,5,5),
          nn.ReLU(),
          nn.Conv2d(5,10,13),
          nn.ReLU(),
		  nn.Conv2d(10,20,5),
		  nn.ReLU(),
		  nn.Conv2d(20,30,5),
		  nn.ReLU(),
		  Reshape(-1, 4 * 4 * 30),
		  nn.Linear(4 * 4 * 30, 500),
		  nn.ReLU(),
		  nn.Linear(500,10)
        )
```

**Comment by Equationliu** [comment]
I have found the reason,,

```
elif 'Flatten' in (str(layer.__class__.__name__)): 
        return DualReshape(in_f, out_f)
```
